features/networking/pod.feature

```diff
@@ -480,3 +480,39 @@ Feature: Pod related networking scenarios
     | ip route |
   Then the output should contain:
     | <%= cb.pod_cidr %> dev <%= cb.tunnel_inf_name %> |
+    
+  # @author anusaxen@redhat.com
+  # @case_id OCP-26373
+  @admin
+  @destructive  
+  Scenario: Pod readiness check for OVN		
+  Given the env is using "OVNKubernetes" networkType
+  And OVN is functional on the cluster
+  Given I switch to cluster admin pseudo user
+  And I use the "openshift-ovn-kubernetes" project
+  And a pod is present with labels:
+    | l | app=ovnkube-node |
+  #Removing CNI config file from container to check readiness probe functionality
+  When I run the :exec client command with:
+    | pod              | <%= pod.name %>                       |
+    | c                | ovnkube-node                          |
+    | oc_opts_end      |                                       |
+    | exec_command     | rm                                    |
+    | exec_command_arg | /etc/cni/net.d/10-ovn-kubernetes.conf |
+  Then the step should succeed
+  #Deleting ovnkube-pod will force CNO to rewrite the conf file and bring cluster back to normal after scenario
+  And I register clean-up steps:
+  """
+  admin ensure "<%= pod.name %>" pod is deleted from the "openshift-ovn-kubernetes" project
```

Comment:
If you use `admin ensure "<%= pod.name %>" pod is deleted from the "openshift-ovn-kubernetes" project after scenario`, then we do not need the step `I register clean-up steps`

Reply:
Yep, aware of that but i actually want ```And OVN is functional on the cluster``` step at very last immediately post after this pod deletion step to make sure i leave cluster in healthy state so i put them under register back to back. WDYT

Reply:
Nevermind, adjusted it in more sensible way now http://pastebin.test.redhat.com/853969.

Reply:
I saw this. Original version also was good. In the new way, better use verification steps as I commented.